### ISSUE TITLE
BALAMT NumberFormatException

### ISSUE THREAD

**Issue by evandrogrm** [stoicflame/ofx4j#32]
When `<BALAMT>` TAG of `LEDGERBAL` contains comma throws a NumberFormatException, other values like `TRNAMT` do not
Error:
`11:40:27,126 ERROR [com.webcohesion.ofx4j.io.AggregateStackContentHandler] (default task-1) Unable to set element 'BALAMT' (property 'amount' of aggregate com.webcohesion.ofx4j.domain.data.common.BalanceInfo): java.lang.NumberFormatException: For input string: "-366622,72"`

**Comment by evandrogrm** [comment]
I found the error, it's in `DefaultStringConversion` from Double type,
changing
```
else if ((Double.class.isAssignableFrom(clazz)) || (Double.TYPE == clazz)) {
  return (E) new Double(Double.parseDouble(value));
}
```
to
```
else if ((Double.class.isAssignableFrom(clazz)) || (Double.TYPE == clazz)) {
  return (E) new Double(Double.parseDouble(value.replace(",", ".")));
}
```
Should fix it

**Comment by stoicflame** [comment]
If you put together a pull request for me, I'll merge it.

**Comment by evandrogrm** [comment]
I did it, I don't know how to merge it. #33

**Comment by stoicflame** [comment]
Fixed in 1.16.
